README.md

```diff
@@ -311,6 +311,31 @@ sudo ln -s $(which rustc)
 
 This step needs to be executed each time Xcode updates.
 
+#### Building using Conda on MacOS
+
+We provide a Conda env for easy Setup. 
+Prequisits: 
+- Have miniconda installed. 
+
+```bash 
+$ conda env create -f env.yml
+$ conda activate VPN
+```
+ - Set a MacOS-SDK (via xcode, or the interwebs) 
```

Comment:
> (via xcode, or the interwebs)

What does this mean? Aren't we setting via setting the `SDKROOT` environment variable?

Reply:
Get! i meant get ,___,